src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health-pie/health-pie.component.ts

```diff
@@ -25,24 +25,41 @@ export class HealthPieComponent implements OnChanges, OnInit {
   @ViewChild('chartTooltip')
   chartTooltipRef: ElementRef;
 
+  readonly CHART_TYPE_DOUGHNUT = 'doughnut';
+  readonly CHART_TYPE_PIE = 'pie';
+  readonly chartTypes = [this.CHART_TYPE_DOUGHNUT, this.CHART_TYPE_PIE];
+
+  readonly COLOR_RED = '#FF6384';
```

Comment:
Isn't there a color scheme we can use instead of defining the colors?
It seems we are reverting a really recent change...

Reply:
The Red & Blue colors were explicitly requested so I set the first ones (next are automatically chosen).
Nevertheless, I've set this topic to be discussed/solved in this issue:
https://tracker.ceph.com/issues/27050